.github/workflows/auto-approve-dependabot-prs.yml

```diff
@@ -0,0 +1,19 @@
+name: Auto-approve Dependabot Pull Requests
+
+on: [pull_request_target]
```

Comment:
Is there any way to have this run after the check suite has run successfully on a pull request?  Maybe a combo of `on` and `if`?  Want to avoid a false sense of approval when a changes has failing CI tests.  I am interested in seeing the effects of this change.  Not sure there will be any.  Could end up with PRs that are approved but need merging/rebasing.

Reply:
We _could_ add a `github-script` step that simply polls the current PR's checks, and that succeeds only once all those checks have succeeded.

But that would mean a bit more hassle if a PR failed for bogus reasons (such as network problems, something that happens only rarely, but it does happen)...